File: lab2(b_file)/main.cpp
#include <iostream>
#include "header.h"
using namespace std;
int main(){
    string fileName1 = "File1.dat";
    string fileName2 = "File2.dat";
    string fileName3 = "m.dat";
    cout <<fileName1 <<":"<<endl;
    getLines(fileName1);
    vector<Ampoule> info = file_text(fileName1);
    cout <<fileName2 <<":"<<endl;
    secondFileLines(fileName2 ,info);
    cout <<fileName1<<":\n";
    outFIle(fileName1);
    cout <<fileName2<<":\n";
    outFIle2(fileName2);
    deleteAmpule(fileName2 ,fileName3);
    cout <<"Modified file 2 :"<<endl;
    outFIle2(fileName2);
}

File: lab2(b_file)/main.py
from foo import *

fileName1 = "File1.dat"
fileName2 = "File2.dat"
fileName3 = "gg.dat"
size = file1(fileName1)
listFile1 = readFile1(fileName1, size)
size2 = file2(fileName2, listFile1)
outFile1(fileName1, size)
outFile2(fileName2, size2)
size3 = removeAmpoule(fileName2, fileName3, size2)
outFile2(fileName2, size3)
